content/5_Plat/Sqrt.mdx

```diff
@@ -119,42 +210,143 @@ recalculate prefix sums.
 <CPPSection>
 
 ```cpp
-//CodeSnip{CPP Short Template}
+#include <bits/stdc++.h>
+using namespace std;
 
 int n, q;
-vi x;
-vector<long long> cum;
+vector<int> arr;
+vector<long long> prefix;
 
+/** Build the prefix array for arr */
 void build() {
-	cum = {0};
-	for(const auto &t : x) cum.pb(cum.back() + t);
+	prefix = {0};
```

Comment:
It isn't good practice to `clear` a vector every time and use `push_back` to reinitialize it, especially when its essentially the same to use a normal loop.